site/docs/releases.md

```diff
@@ -64,28 +64,36 @@ High-level features:
 
 * **Core**
   * Allow Iceberg schemas to specify one or more columns as row identifers [[\#2465](https://github.com/apache/iceberg/pull/2465)]. Note that this is a prerequisite for supporting upserts in Flink.
-  * Added JDBC [[\#1870](https://github.com/apache/iceberg/pull/1870)] and DynamoDB [[\#2688](https://github.com/apache/iceberg/pull/2688)] Iceberg catalog implementations.
+  * Added JDBC [[\#1870](https://github.com/apache/iceberg/pull/1870)] and DynamoDB [[\#2688](https://github.com/apache/iceberg/pull/2688)] catalog implementations.
   * Added predicate pushdown for partitions and files metadata tables [[\#2358](https://github.com/apache/iceberg/pull/2358), [\#2926](https://github.com/apache/iceberg/pull/2926)].
   * Added a new, more flexible compaction action for Spark that can support different strategies such as bin packing and sorting. [[\#2501](https://github.com/apache/iceberg/pull/2501), [\#2609](https://github.com/apache/iceberg/pull/2609)].
   * Added the ability to upgrade to v2 or create a v2 table using the table property format-version=2  [[\#2887](https://github.com/apache/iceberg/pull/2887)].
```

Comment:
Core: Support nulls in StructLike collections (#2929)

Reply:
Core: Fix partition field IDs in table replacement (#2906)

Reply:
Core: Add check-status config to TableProperties (#2596)

Note: This probably needs more context since it's intended to help address issues with Metastore timeouts, but is not clear from the commit message.

Reply:
Yeah, that's another one I flagged. The original commit is #2328 where we introduced a new exception to signal that commit state was uncertain and status checks. There were a few follow-ups to that as well, including this one that adds configuration for the checks, and support for the exception in Glue (#2402) and Nessie (#2515).

I think this deserves a high-level callout because it is a big fix that a lot of people are waiting on. I think Shopify was waiting for it, for example.

Reply:
I'm not sure I'd include #2929 since that was extending an internal class for another use rather than fixing a user-facing problem. Up to you though.

Reply:
Also: Added key_metadata to manifest lists for encryption (#2675)

Reply:
@danielcweeks, can you please suggest a blurb for @2596?

Reply:
I added blurbs for #2929, #2906, #2675. Waiting on @danielcweeks for a blurb about #2596.